includes/documents/class-wcpdf-invoice.php

```diff
@@ -441,7 +441,7 @@ public function get_pdf_settings_fields( $option_name ) {
 				'args'			=> array(
 					'option_name'	=> $option_name,
 					'id'			=> 'invoice_number_search',
-					'description'   => __( 'Can potentially slow down the search process.', 'woocommerce-pdf-invoices-packing-slips' ),
+					'description'   => __( 'Can potentially slow down the search process. You can also use the <a href="https://woocommerce.com/document/high-performance-order-storage/" target="_blank">HPOS</a> feature to search orders by invoice numbers using search type selector, which is faster than this option.', 'woocommerce-pdf-invoices-packing-slips' ),
```

Comment:
I think this setting can be disabled and omitted when running HPOS.

Reply:
Actually, you could rely on it to enable the search in HPOS too?

Reply:
@alexmigf I don't understand this completely. You think that I should only show Invoice Number in HPOS selector in case this option is enabled?

Reply:
Yes

Reply:
@alexmigf I agree. I have also modified this option's description so that it is not confusing for users.